tests/components/filter_menu.test.ts

```diff
@@ -201,6 +201,18 @@ describe("Filter menu component", () => {
       ]);
     });
 
+    test("Hitting esc key correctly closes the filter menu", async () => {
+      createFilter(model, "A1:A2");
+      await nextTick();
+      await simulateClick(".o-filter-icon");
+      expect(fixture.querySelectorAll(".o-filter-menu")).toHaveLength(1);
+      document
+        .querySelector(".o-grid")!
+        .dispatchEvent(new KeyboardEvent("keydown", { key: "Escape", bubbles: true }));
+      await nextTick();
```

Comment:
use `keyDown` helper here will not pass the tests, but using these lines will. Don't know why (the same thing happens for the filter menu in the `grid.test.ts`, too )